Dockerfile

```diff
@@ -2,10 +2,13 @@ FROM centos:centos7
 
 EXPOSE 8080
 
+RUN curl --retry 999 --retry-max-time 0 -sSL https://github.com/openshift/origin/releases/download/v1.2.0/openshift-origin-client-tools-v1.2.0-2e62fab-linux-64bit.tar.gz | tar xzv && \
+  mv openshift-origin-*/* /usr/bin/
```

Comment:
@rhcarvalho What's the best practise with multiple RUN instructions in a Dockerfile? Is it best to keep them separate producing more layers, or bundle them all up in one RUN instruction? i.e Should this be added to the one below?